src/log_outgoing_requests/log_requests.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from requests import Session
+
+logger = logging.getLogger("requests")
+
+
+def hook_requests_logging(response, *args, **kwargs):
+    """
+    A hook for requests library in order to add extra data to the logs
+    """
+    extra = {"requested_at": timezone.now(), "req": response.request, "res": response}
+    logger.debug("External request", extra=extra)
```

Comment:
For consistency this message string should also be 'Outgoing' instead of 'External' (this applies everywhere from the old name)